SettingsSerializer.cpp

```diff
@@ -23,78 +23,773 @@ std::string SettingsSerializer::SetGeneralSettings(const unsigned int* captureFr
                                            const bool* startOnTrigSoftware, const EProcessingActions* processingActions,
                                            const EProcessingActions* rtProcessingActions, const EProcessingActions* reprocessingActions)
 {
-    return mSerializer->SetGeneralSettings(captureFrequency, captureTime, startOnExtTrig, startOnTrigNO, startOnTrigNC, startOnTrigSoftware, processingActions, rtProcessingActions, reprocessingActions);
+    auto theGeneral = mSerializer
```

Comment:
nitpick: weird line splitting here.

Reply:
I see now that this is done in several places. Do we think this increases readability?